README.md

```diff
@@ -36,7 +36,17 @@ It will also set up:
 
 ## Set up
 
-First, install cookiecutter in your desired environment.
+### Installating Cookiecutter
+
+First, install cookiecutter in your desired environment. Running in the terminal in your environment, with Pip:
+
+`pip install cookiecutter`
+
+or conda:
+
+`conda install cookiecutter`
```

Comment:
Should this come from the main channel or conda-forge?

Reply:
@JoeZiminski bump

Reply:
should come from conda forge, is:

`conda install -c conda-forge cookiecutter`

or

```
conda config --add channels conda-forge
conda install cookiecutter
```
better to recommend?

Reply:
my preference would be the former. While the latter is more explicit, my expectation is that advanced users are familiar with the former, and new users of `conda` won't really understand either, so we should give them one fewer line to copy-paste?

Reply:
No idea really. Maybe we want to say `conda install -c conda-forge cookiecutter`. Here probably isn't the place to start a lesson about conda channels?